src/style.rs

```diff
@@ -11,6 +11,10 @@ pub enum StyleComponent {
     Grid,
     Rule,
     Header,
+    Filename,
```

Comment:
Just so I don't forget: We should consider marking this enum as `#[non_exhaustive]`, similar to #2012